src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -4,7 +4,6 @@
 
 public class ColorSupplier {
     public String getRandomColor() {
-        int colorIndex = new Random().nextInt(Colors.values().length);
-        return Colors.values()[colorIndex].toString();
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
You can pass the existing instance to the constructor. Because otherwise, each time you call this method, a new instance will be created.

Reply:
I really don't understand how I'm supposed to do it. Could you help me?